website/docs/docs/cloud/manage-access/enterprise-permissions.md

```diff
@@ -111,6 +111,7 @@ Job admin is an administrative permission set for users who create, run, and man
 Notable features:
 - Job admin is a project-level set.
 - Job admins can create and edit jobs, runs, environment variables, and data warehouse configs. 
+- Job admins can set up project integrations, including [Tableau lineage](/docs/cloud-integrations/semantic-layer/tableau).
```

Comment:
do we mean downstream exposures? the current link is for the SL tableau integration to query metrics
```suggestion
- Job admins can set up project integrations, including [Tableau lineage](/docs/cloud-integrations/downstream-exposures-tableau).
```